### PR TITLE
Update eslint.config.js

### PR DESCRIPTION
**Pull request by SaurabhKhare86**
### Summary
Updated eslint config for consistency. In all places it will use `off`, `warn` and `error` instead of `0`, `1` and `2`.

### Additional Details
Just updated config to use string values instead of numbers

### CODE REVIEW

**Comment by mjhenkes** [comment]
@SaurabhKhare86 Thanks for the contribution! Could you update the change log as part of this? https://github.com/SaurabhKhare86/terra-toolkit/blob/patch-1/packages/eslint-config-terra/CHANGELOG.md

**Review by emilyrohrbough — APPROVED** [review]
Looks good with changelog addition. Thank you!

**Review by StephenEsser — APPROVED** [review]

**Comment by SaurabhKhare86** [comment]
Updated changelog

File: packages/eslint-config-terra/eslint.config.js [hunk 1]
```diff
@@ -24,15 +24,15 @@ module.exports = {
     // Replaces jsx-a11y/label-has-for rule. By default, it wants inputs to be both wrapped in a label
     // and include a id/for attribute mapping with label.
     // This config updates the rule to require one or the other.
-    'jsx-a11y/label-has-associated-control': [2, { assert: 'either' }],
-    'no-multiple-empty-lines': [1, {
+    'jsx-a11y/label-has-associated-control': 'error', { assert: 'either' }],
```
Comment (mjhenkes) [inline packages/eslint-config-terra/eslint.config.js]:
```suggestion
    'jsx-a11y/label-has-associated-control': ['error', { assert: 'either' }],
```

**Comment by mjhenkes** [comment]
@SaurabhKhare86, sorry, one last thing. Could you add yourself as a contributor? https://github.com/SaurabhKhare86/terra-toolkit/blob/patch-1/CONTRIBUTORS.md

**Comment by SaurabhKhare86** [comment]
@mjhenkes  Updated contributors and added missing opening square bracket (Glad you spotted that).

**Review by benbcai — APPROVED** [review]

**Review by mjhenkes — APPROVED** [review]

**Pull request merged**